### DISCUSSION TITLE
🚀 Feature: Ability to export and import project to one appwrite instance to another.

### DISCUSSION THREAD

**Discussion by usamashafiqs** [appwrite/appwrite#4956] [Q&A]
### 🔖 Feature description

Ability to export and import project to one appwrite instance to another.

### 🎤 Pitch

I have 4 project in my appwrite instance i wanna move 2 project to another appwrite instance or new appwrite instance.
Firebase had option to export database in single json file and export storage in zip file.
I can't find any export and import options in appwrite.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
The easiest way to migrate a schema is using the [Appwrite CLI](https://appwrite.io/docs/command-line#initializeProject). Just so you know, the CLI currently wipes a collection's existing attributes and indexes before deploying which would result in data loss. We plan to improve this in the future.

**Reply by usamashafiqs** [reply]
@stnguyen90 There is no way to migrate without data loss ?

**Reply by stnguyen90** [reply]
The data loss would only occur if you have any data in the target instance. As I said, we plan to improve the CLI in the future.

The other alternative would be for you to write migration scripts yourself that wouldn't delete attributes.

**Comment by tripolskypetr** [comment]
It looks like the dev team are not going to add that feature because they don't want to give you ability to migrate from appwrite to better open source solution. [The similar issue](https://github.com/appwrite/appwrite/issues/3050) was closed too, I expect they simple mark this discussion as solved
